torchtitan/config_manager.py

```diff
@@ -194,6 +194,15 @@ def __init__(self):
             action="store_true",
             help="Whether the fused implementation(CUDA only) is used.",
         )
+        self.parser.add_argument(
+            "--optimizer.backward",
+            type=bool,
```

Comment:
we can use `action="store_true",` since it's a boolean flag. Please check other configs as examples.